### PR TITLE
[SMALLFIX] Clean up in underfs and test module

### PR DESCRIPTION
**Pull request by HorizonNet**
Small clean up of some classes in the *underfs* and *test* package.

RPCMessageIntegrationTest:
- Simplified return statement

AlluxioShellUtilsTest:
- Removed explicit argument types

LocalMiniDFSCluster:
- Removed unnecessary cast

BlockServiceHandlerIntegrationTest:
- Made a field a local variable

GCSUnderFileSystem:
- Removed explicit argument type

### CODE REVIEW

File: tests/src/test/java/alluxio/underfs/hdfs/LocalMiniDFSCluster.java [hunk 1]
```diff
@@ -232,7 +232,7 @@ public void start() throws IOException {
 
       // For HDFS of earlier versions, getFileSystem() returns an instance of type
       // {@link org.apache.hadoop.fs.FileSystem} rather than {@link DistributedFileSystem}
-      mDfsClient = (DistributedFileSystem) mDfsCluster.getFileSystem();
+      mDfsClient = mDfsCluster.getFileSystem();
```
Comment (apc999) [inline tests/src/test/java/alluxio/underfs/hdfs/LocalMiniDFSCluster.java]: Here the conversion is intentional and we should keep it. See the comment for the reason.

**Comment by apc999** [comment]
I left one comment for the conversion. Otherwise it looks all good to me. Thanks.

**Comment by HorizonNet** [comment]
@apc999 Thanks for the comment. I updated the PR accordingly.

**Comment by apc999** [comment]
Thanks for the update.
LGTM

**Pull request merged**